.github/workflows/tests.yml

```diff
@@ -19,9 +19,10 @@ jobs:
 
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         os: [ 'ubuntu-latest' ] #, macos-latest, windows-latest ]
-        php-version: [ '7.3', '7.4', '8.0', '8.1', '8.2' ]
+        php-version: [ '8.0', '8.1', '8.2' ]
```

Comment:
```suggestion
        php-version: [ '8.0', '8.1', '8.2', '8.3' ]
```

Reply:
Would also need an update to composer.json.

Reply:
See GH-130.